docs/AR.md

```diff
@@ -0,0 +1,92 @@
+# Augmented Reality Documentation
+
+![Viro](https://raw.githubusercontent.com/ViroCommunity/virocommunity.github.io/main/viro_community_logo.png)
+
+The information required to use the Augmented Reality feature will be specified here
+
+## Package used and system requirements
+
+- [Viro](https://viro-community.readme.io)
+- [iOS Requirements](https://developer.apple.com/library/archive/documentation/DeviceInformation/Reference/iOSDeviceCompatibility/DeviceCompatibilityMatrix/DeviceCompatibilityMatrix.html)
+- [Android Requirements](https://developers.google.com/ar/devices)
+
+## What to do on the server side
+
+- Go to the CMS page and log in
+- Go to the list of tours
+- Select the tour you want to add Augmented Reality feature to
+- Add new Tourstopps on the tour detail page
+- Edit the sections added for AugmentedReality in Tourstopps
+- To find out what types of files you can upload, see the next section
+
+## What to do on the server side
```

Comment:
is it intended to use this headline twice?

Reply:
Sorry, I missed it.
https://github.com/ikuseiGmbH/smart-village-app-app/pull/659/commits/c9fb53e0228b5c377b6f185e594d535588212412